plugins/rbac-backend/src/helper.test.ts

```diff
@@ -0,0 +1,249 @@
+import { RoleMetadataDao } from './database/role-metadata';
+import {
+  deepSortedEqual,
+  metadataStringToPolicy,
+  policiesToString,
+  policyToString,
+  removeTheDifference,
+  transformArrayToPolicy,
+} from './helper'; // Import the function to test
```

Comment:
Looks like you are failing the prettier check, you can fix it by running `yarn prettier:fix`